en/micro-integrator/docs/references/connectors/redis-connector/redis-connector-configuration.md

```diff
@@ -0,0 +1,36 @@
+# Setting up the Environment 
+
+The Redis connector allows you to access the Redis commands through the WSO2 EI. Redis stands for remote dictionary server. Redis store /serve that stores data as key-value pairs and this key-value store can be used as a database.
+
+## Setting up the environment
+
+Before you start configuring the Redis connector, you also need WSO2 MI, and we refer to that location as <PRODUCT_HOME>.
```

Comment:
Let's not use the name `WSO2 MI` anywhere. It is actually EI 7. Thus we'll just mention WSO2 EI. There is no need to say `also`. 

Download WSO2 EI (link) and extract the zip file to a known location. In this setup guide we refer to that location as <PRODUCT_HOME>.

Reply:
Updated.